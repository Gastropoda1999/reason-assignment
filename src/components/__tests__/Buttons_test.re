open Jest;
open Expect;

let _ =

describe("Buttons", () => {
  test("renders", () => {
    let dispatch = _action => ();
    let tree = <Buttons dispatch /> |> ReactShallowRenderer.renderWithRenderer;

    expect(tree) |> toMatchSnapshot;
  });
});
