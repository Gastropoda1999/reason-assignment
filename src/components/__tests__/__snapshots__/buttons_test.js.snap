// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Buttons renders 1`] = `
<div>
  <ButtonGroup
    reasonProps={
      Array [
        Array [
          "ButtonGroup",
          [Function],
          Array [
            0,
          ],
          [Function],
          [Function],
          [Function],
          [Function],
          [Function],
          [Function],
          [Function],
          [Function],
          0,
          [Function],
          0,
        ],
      ]
    }
  />
  <ButtonGroup
    reasonProps={
      Array [
        Array [
          "ButtonGroup",
          [Function],
          Array [
            0,
          ],
          [Function],
          [Function],
          [Function],
          [Function],
          [Function],
          [Function],
          [Function],
          [Function],
          0,
          [Function],
          0,
        ],
      ]
    }
  />
  <ButtonGroup
    reasonProps={
      Array [
        Array [
          "ButtonGroup",
          [Function],
          Array [
            0,
          ],
          [Function],
          [Function],
          [Function],
          [Function],
          [Function],
          [Function],
          [Function],
          [Function],
          0,
          [Function],
          0,
        ],
      ]
    }
  />
  <ButtonGroup
    reasonProps={
      Array [
        Array [
          "ButtonGroup",
          [Function],
          Array [
            0,
          ],
          [Function],
          [Function],
          [Function],
          [Function],
          [Function],
          [Function],
          [Function],
          [Function],
          0,
          [Function],
          0,
        ],
      ]
    }
  />
  <ButtonGroup
    reasonProps={
      Array [
        Array [
          "ButtonGroup",
          [Function],
          Array [
            0,
          ],
          [Function],
          [Function],
          [Function],
          [Function],
          [Function],
          [Function],
          [Function],
          [Function],
          0,
          [Function],
          0,
        ],
      ]
    }
  />
</div>
`;
